app/models/user.rb

```diff
@@ -1,12 +1,17 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
Please be sure that the application asks for confirmation of email as it is part of the project requirements 📝 

Your model should look something like:
```ruby
devise :database_authenticatable, :registerable,
				:recoverable, :rememberable, :validatable, :confirmable
```

Reply:
Thank you @mateo951. I added the feature.